### DISCUSSION TITLE
`:h coc` or `:h coc-nvim` returns nothing

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#3932] [Q&A] [answered]
[This file](https://github.com/neoclide/coc.nvim/blob/master/doc/coc.txt) is not available in the help?

**Comment by fannheyward** [comment]
Can't reproduce, `:h coc-nvim` works as expected. How do you install coc.nvim?

**Reply by vitaly-zdanevich** [reply]
Vim 8 without package manager, Gentoo Linux.

Git clone to `~/.vim/pack/git-submodules-plugins/start/`

On first start - some command inside Vim to install dependencies.

**Reply by fannheyward** [reply]
Run `:helptags ALL` and try again.

**Reply by vitaly-zdanevich** [reply]
Thanks, this works - maybe install script may include this.

**Comment by chemzqm** [comment, marked as answer]
You have to generate tag files.

**Reply by vitaly-zdanevich** [reply]
`:helptags ALL` did the job.
